src/ShallowTraversal.js

```diff
@@ -125,7 +124,7 @@ export function buildPredicate(selector) {
         return node => nodeMatchesObjectProps(node, selector);
       }
       throw new TypeError(
-        'Enzyme::Selector does not support an array, null, or empty object as a selector',
+        'Enzyme::Selector does not support an array, null, or empty object as a selector'
```

Comment:
I'm not sure what happened but it seems that I used some kind of auto-format that I'm not aware of... sorry for that.

Correcting that !